dev/com.ibm.ws.security.saml.sso_fat.config/fat/src/com/ibm/ws/security/saml/sso/fat/config/SAMLMisc1ConfigTests.java

```diff
@@ -900,7 +900,7 @@ public void test_config_loginPageURL_invalid() throws Exception {
      * time (not having to reconfig multiple times) we'll do all of this in one
      * test)
      */
-    @AllowedFFDC(value = { "com.ibm.ws.security.saml.error.SamlException", "org.opensaml.ws.security.SecurityPolicyException", "org.opensaml.xml.signature.SignatureException" })
```

Comment:
Similarly AllowedFFDC could be made Repeatable to allow FFDCs for certain exceptions for certain repeat actions.

Reply:
I will address this fattest.simplicity update and repeatable AllowedFFDC changes with another issue/pr